File: javascript/3048.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var n = parseInt(lines.shift()), total = 0, b = 3

for(let i = 0; i < n; i++) {
    var a = parseInt(lines.shift())

    if(b !== a) {
        b = a
        total++
    }
}
console.log(total)

File: C++/3048.cpp
#include <iostream>

using namespace std;

int main(){
    int n, i, a, b = 3, total = 0;
    cin >> n;
    
    for(i = 0; i < n; i++) {
        cin >> a;
        
        if(b != a) {
            b = a;
            total++;
        }
    }

    cout << total << endl;
    return 0;
}